src/sage_docbuild/conf.py

```diff
@@ -1038,7 +1047,11 @@ def apply(self):
                         content = Container("", is_div=True, classes=["tab-content"])
                         content += cell_node
                         container += content
-                        parent.insert(index + 1, container)
+                        parent.insert(index, container)
+                        index += 1
+                        if isinstance(prev_node, nodes.paragraph):
+                            prev_node['classes'].append('with-sage-live-tab')
+
 
```

Comment:
there is an extra blank line inserted at L1054.

Reply:
Pushed a fix. Sorry I missed that.

Reply:
Thanks.

Don't be sorry. We all make mistakes. One reason why we have the reviewer system.